### PR TITLE
Tag Distributions v0.10.1

### PR DESCRIPTION
**Pull request by ararslan**

### CODE REVIEW

File: Distributions/versions/0.10.1/sha1 [hunk 1]
```diff
@@ -0,0 +1 @@
+a71f7b48d2feb554459491039d4ca7da3384252e
```
Comment (tkelman) [inline Distributions/versions/0.10.1/sha1]: this commit is identical to 0.10.0

Comment (ararslan) [inline Distributions/versions/0.10.1/sha1]: ...what. Okay, I did something weird and I'm not entirely sure what.

Comment (tkelman) [inline Distributions/versions/0.10.1/sha1]: tagged without checkout first?

Comment (ararslan) [inline Distributions/versions/0.10.1/sha1]: Nope, did the usual checkout master, tag, publish...

Comment (ararslan) [inline Distributions/versions/0.10.1/sha1]: `Pkg.tag` should have a check for when the tag you're adding is identical to an existing one.

Comment (tkelman) [inline Distributions/versions/0.10.1/sha1]: yep, open an issue in PkgDev?

**Comment by tkelman** [comment]
This looks like it broke a lot of dependent packages, and should probably have been at least a minor bump - http://pkg.julialang.org/pulse.html

To fix this short-term, what would you think about creating a branch from right before the major rewrite, re-applying the deprecation fixes and tagging that as 0.10.2?

**Pull request merged**